network/api.go

```diff
@@ -192,7 +208,7 @@ func buildNetworkDebugInfoData(debugConns []*Connection) string {
 
 func (conn *Connection) debugInfoLine() string {
 	var connectionData string
-	if conn.ID != "" {
+	if conn.Type == DNSRequest { // conn.ID !=
```

Comment:
I guess the comment at the end of the line can be removed

Reply:
Already fixed in previous commit, you seem to have review an old state, before some small fixes.